Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -1832,18 +1832,35 @@ def scan_url_command(client: Client, args: dict) -> CommandResults:
     1 API Call
     """
     url = args['url']
-    raw_response = client.url_scan(url)
-    data = raw_response['data']
-    data['url'] = url
-    context = {
-        f'{INTEGRATION_ENTRY_CONTEXT}.Submission(val.id && val.id === obj.id)': data,
-        'vtScanID': data.get('id')  # BC preservation
-    }
+    raw_response: Dict[str, Any] = {}
+    data: Dict[str, Any] = {}
+    context: Dict[str, Any] = {}
+    headers = ['id', 'url']
+
+    try:
+        raw_response = client.url_scan(url)
+        data = raw_response['data']
+
+        data['url'] = url
+        context = {
+            f'{INTEGRATION_ENTRY_CONTEXT}.Submission(val.id && val.id === obj.id)': data,
+            'vtScanID': data.get('id')  # BC preservation
+        }
+    except DemistoException as ex:
+        error = ex.res.json().get('error')
+
+        # Invalid url, probably due to an unknown TLD
+        if error['code'] == 'InvalidArgumentError':
+            data = {'url': url, 'id': '', 'error': error['message']}
+            headers.append('error')
+        else:
+            raise ex
```

Comment:
for keeping the original stack trace
```suggestion
            raise
```